### PR TITLE
Form prioritization wg

### PR DESCRIPTION
**Pull request by spastorino**
cc @pnkfelix @nikomatsakis @wesleywiser

### CODE REVIEW

File: content/working-groups/prioritization/_index.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: Prioritization Working Group
+type: docs
+---
+
+# Prioritization Working Group
+![working group status: active][status]
+
+This working group aims to make the compiler easier to learn by ensuring that rustc-dev-guide and api docs are "complete".
```
Comment (bjorn3) [inline content/working-groups/prioritization/_index.md]: Copy paste?

Comment (spastorino) [reply content/working-groups/prioritization/_index.md]: Doh, fixed.

File: content/working-groups/prioritization/FAQ.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+title: FAQs
+type: docs
+---
+# Frequently Asked Questions (FAQ)
+
+If you have a question, feel free to file an issue or ask in the working group's Zulip stream.
+
+There are no FAQs so far. Please ask us more questions.
```
Comment (nikomatsakis) [inline content/working-groups/prioritization/FAQ.md]: we probably don't need this file here -- I think the template should be slimmed down

File: content/working-groups/prioritization/_index.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+## What is the goal of this working group?
+
+This working group aims to accomplish the following:
+
+- Processing 'nominations' and routing bugs to folks who can fix them
+- Identifying *critical* bugs and monitoring them to ensure they are making progress
+- Identifying the agenda for compiler team triage meetings
+  - Critical issues that are not making progress
+  - Issues where bugs are nominated for needing wider discussion
+    - Ideally, crystallize
```
Comment (nikomatsakis) [inline content/working-groups/prioritization/_index.md]: incomplete sentence

File: content/working-groups/prioritization/_index.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+- Tracking deferred things and ensuring they are picked up again
+  - Future compatibility warnings
+
+# How can I get involved?
+
+If you are interested in getting involved in this working group, come and say hi in the Zulip
+stream. You can be added to the Zulip group for the working group if you are interested in being
+pinged when there are available tasks.
```
Comment (nikomatsakis) [inline content/working-groups/prioritization/_index.md]: Maybe here the answer is "try to join the meeting" more than anything?

Comment (nikomatsakis) [reply content/working-groups/prioritization/_index.md]: I think we should talk about the meeting and link to the compiler team calendar here

File: layouts/shortcodes/checkin-schedule.html [hunk 5]
```diff
@@ -29,6 +29,7 @@
         "Parallel Rustc",
         "Polonius",
         "Polymorphization",
+        "Prioritization",
```
Comment (nikomatsakis) [inline layouts/shortcodes/checkin-schedule.html]: Do we need/want check-ins from prioritization wg?

File: content/working-groups/prioritization/FAQ.md (hunk 2 above)
Comment (spastorino) [reply content/working-groups/prioritization/FAQ.md]: Ok going to remove this file.

File: content/working-groups/prioritization/_index.md (hunk 3 above)
Comment (spastorino) [reply content/working-groups/prioritization/_index.md]: Yeah, hehe I had the same thought when I copied that from the Zulip conversation. Going to remove this.

File: content/working-groups/prioritization/_index.md (hunk 4 above)
Comment (spastorino) [reply content/working-groups/prioritization/_index.md]: Done!.

File: layouts/shortcodes/checkin-schedule.html (hunk 5 above)
Comment (spastorino) [reply layouts/shortcodes/checkin-schedule.html]: I had the same doubt but ended thinking well if there's nothing to say we can skip the checkin but there may be changes to share like ... we've changed labels in this or that way, we are doing our prioritization async, at the beginning we had 50 P-high issues we now have 10 and stuff like that :). Probably a lot of things could happen at the beginning I guess long term we should remove this from here. Still not 100% sure, let me know what you think.

Comment (wesleywiser) [reply layouts/shortcodes/checkin-schedule.html]: This is the right thing to do per the working group formation guidelines but I wonder if perhaps we should consider changing that. Adding this wg will bring us to 13 wgs which means, if I've done my math correctly, we'll only hear from a wg every 1.5 months. That seems like a fairly long time to find out if a wg is stalled or blocked on something and not making progress (especially if that wg is doing work that's part of our yearly goals). 

Perhaps we should consider only requiring checkins from teams working toward our yearly goals? Other teams can of course make use of the announcement time at the beginning of the weekly compiler team meeting to report updates on anything happening. 

(I don't feel strongly about this, it's just an idle thought I had.)

Comment (spastorino) [reply layouts/shortcodes/checkin-schedule.html]: I have no clear idea either. Another possibility is to do 3 checkins per week. Mainly given that from time to time there are some wgs with no updates to share.

**Comment by spastorino** [comment]
@nikomatsakis saw that you've merged the team PR. This one is ready IMO but unsure what do you prefer related to checkins.

File: layouts/shortcodes/checkin-schedule.html (hunk 5 above)
Comment (nikomatsakis) [reply layouts/shortcodes/checkin-schedule.html:32]: I think let's merge as is and discuss changing how we handle check-ins later. I also don't actually think every 1.5 months is that bad, but I do think we can do better.

**Pull request merged**